### PR TITLE
Added build_simple script and Dockerfile for building inside docker-container

### PR DESCRIPTION
**Pull request by sea-kg**
- build_simple.sh - file with cmake commands and comment with requirements
- Dockerfile - for building sources inside docker-container (prepared building enviroment)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# @v0.0.1
+FROM sea5kg/wsjcpp:latest
```
Comment (fenugrec) [inline Dockerfile:2]: As I understand, this is the "parent image" you're building upon ? can we use something more generic / public ?

**Comment by fenugrec** [comment]
Interesting PR, thanks for submitting. Is the `build_simple.sh` required for the docker or is it just an example ?

**Comment by sea-kg** [comment]
Ok.

Maybe better add HOW-TO-BUILD.md  in a root of project ?

**Comment by fenugrec** [comment]
> Maybe better add HOW-TO-BUILD.md in a root of project ?

No, I don't think I want another file. I may add a paragraph in the README that mentions `build-simple.sh`, but it already references `doc/build_system.txt` which has detailed information. I don't want compiling instructions in 3 different places...

**Comment by sea-kg** [comment]
Ok. I m understood.

**Pull request closed without merging**